src/include/local/stir/motion/NonRigidObjectTransformationUsingBSplines.h

```diff
@@ -53,8 +53,16 @@ class NonRigidObjectTransformationUsingBSplines
 public:
   static const char * const registered_name;
 
+  // Default constructor
   NonRigidObjectTransformationUsingBSplines();
 
+  /// Give x, y and z components of the deformation field images separately.
+  /// N.B., this will only work if the three components are in STIR orientation.
+  NonRigidObjectTransformationUsingBSplines(const std::string&, const std::string&, const std::string&, const int bspline_order);
+
+  // Give x, y and z components of the deformation field images together (e.g., multicomponent nifti)
```

Comment:
`///` for doxygen. 
Also best to name the variables here to make the order clear.

Reply:
I intentionally did doxygen, do you not want it?